src/page/basic-page.jsx

```diff
@@ -9,19 +11,31 @@ import Nav from "../component/nav.jsx";
  *   The basic page fields.
  */
 const BasicPage = ({ fields }) => {
+  console.log(window.location.href);
   // Extract the required fields from the resource fields.
-  const { mainMenu, title, body } = fields;
+  const { mainMenu, title, created, relatedLinks, body } = fields;
   return (
     <div id="basic-page">
-      <header>
-        {/* The main resource. */}
-        <Nav resource={mainMenu.data} />
-        <h2>{title}</h2>
-      </header>
+      <Header menu={mainMenu}></Header>
       <main>
-        <section>
-          {/* The body content is sent from the server, pre-processed and filtered against XSS vulnerabilities. */}
-          <ProcessedHTML html={body} />
+        <section className="basic-page-content">
+          {relatedLinks && relatedLinks.data.length > 0 && (
```

Comment:
it's a good practice to use a ternary here instead of the `&&`, even if the `:` is `null`
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx